firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -452,20 +453,36 @@ public void updateToNewRelease_receiveProgressUpdateFromUpdateApp() {
   }
 
   @Test
-  public void taskCancelledOnScreenRotation_whenUpdateDialogShowing() {
+  public void signInConfirmationDialogReappears_whenScreenRotates_whenSignInDialogShowing() {
```

Comment:
Nit, can you rename these test names to use the format:

nameOfMethodCalled_conditionsForThisTest_expectedBehavior?

go/unit-testing-practices#naming

Reply:
yea there isn't one method we are calling so wasn't sure what to do with that.

Reply:
Ah yeah, to be clear it's more like the method under test, which in this case would be `updateIfNewReleaseAvailable`.

Reply:
cool i was leaning towards using the basic config since thats what we call. Thanks!